cypress/integration/customTypes/00-create.spec.js

```diff
@@ -1,24 +1,43 @@
 describe("Custom Types specs", () => {
-  const name = "My Test";
-  const id = "my-test";
+  let name = "My Test";
+  let id = "my-test";
+
   beforeEach(() => {
+    const ID = Math.floor(10000 + Math.random() * 90000);
```

Comment:
this is so that the test always creates a CT with a dynamic name. The issue with it being the same (e.g. always "My Test") is that if it creates it but then fails just after, it will always fail on any subsequent re-tries (because you will not be able to create 2 CT with the same name)

Reply:
dos deleting it first not work?